app/controllers/api/application_instances_controller.rb

```diff
@@ -22,12 +22,14 @@ def show
   def create
     @application_instance.domain =
       "#{@application_instance.lti_key}.#{ENV['APP_URL']}"
+    @application_instance.config = params[:application_instance][:config]
```

Comment:
Add a note about why we have to use the param directly instead of using strong params.